### ISSUE TITLE
Deprecate 

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#14638]
## Description

We now have a new [cookie settings](https://www.mozilla.org/en-US/privacy/websites/cookie-settings/) page that controls cookie consent site-wide, so we should deprecate (and redirect) [this older page](https://www.mozilla.org/en-US/privacy/websites/data-preferences/) that was related only to first-party data.

Before we can do that, we need to give the MDN team notice as they are currently using the older page as an opt-out page for Glean.

---

#### Success Criteria
- [ ] Old data preferences page is removed and redirected to `/cookie-settings/`

**Comment by fiji-flo** [comment]
For now MDN relies on the old page for opting out of Glean. Can we keep it, till we migrated?

**Comment by janbrasna** [comment]
Right now the docs and cookie settings mention or expect there's a link from https://www.mozilla.org/privacy/websites/ to the new cookie settings page, however the `privacy/websites` still only links the older data preferences page (i.e. there's currently no link to the new cookie settings page from anywhere unless you come across the cookie banner).

Until this is unblocked, and the link starts redirecting from the old prefs to the new prefs, isn't it an issue the link to the currently used prefs is not there?

I've also noticed the `privacy/websites` text has a few other tools listed that are probably no longer used, so an update to it might actually help clean it up too:

> - Mozilla first-party tools: Opt out on www.mozilla.org.   _(here the link is only to the old prefs)_
> - Convert.com: Opt out on www.mozilla.org.
> - ShareProgress and Fathom.com: You can turn on Do Not Track in your browser.

and also down in metrics:

> - Convert.com and ShareProgress, which place a cookie on your device, to help us test variations of web content. This helps us offer better web experiences to users.

(I thought Convert code is long removed…)

Is this something that needs updating perhaps from legal before it can reflect the tracking used today?

**Comment by alexgibson** [comment]
> (i.e. there's currently no link to the new cookie settings page from anywhere unless you come across the cookie banner).

This is incorrect. There's a "cookies" link to the new settings page in the main footer of every page on mozorg, which is a legal requirement (and available globally). However we can't remove the old privacy link in `/privacy/websites/` until MDN removes it's usage of the old page (hence this issue),. But hopefully, this shouldn't take long (any updates here @fiji-flo?)

> I've also noticed the privacy/websites text has a few other tools listed that are probably no longer used, so an update to it might actually help clean it up too:

Please file issues for legal notices pages over in https://github.com/mozilla/legal-docs. Thanks

**Comment by janbrasna** [comment]
Thanks! I see it's been already reported: https://github.com/mozilla/legal-docs/issues/1987

**Comment by alexgibson** [comment]
I forgot I did that one already 😅

**Comment by stephaniehobson** [comment]
@fiji-flo Has MDN migrated away from using this page for opting out of Glean yet?

**Comment by fiji-flo** [comment]
Hey @stephaniehobson this is still not resolved. I'll increase the pressure.
